model_analyzer/model_manager.py

```diff
@@ -263,21 +263,33 @@ def _create_model_variant(self, original_name, variant_config):
                 pass
 
     def _load_model_variant(self, variant_config):
+        """
+        Decides on whether or not to load a model variant in the client
+        """
+        retval = True
+        if self._config.triton_launch_mode == 'remote':
+            # Don't load model if in remote and reload disabled
+            reload_model = not self._config.reload_model_disable
+            if reload_model:
+                retval = self._do_load_model_variant(variant_config)
```

Comment:
I think this would read cleaner if there was only a single call to `_do_load_model_variant`.  Make a local boolean `do_load` based on launch_mode and disable, and then guard the single call with that bool.

Reply:
Done.